fastai/optimizer.py

```diff
@@ -44,17 +68,23 @@ def set_hyper(self, k, v):
         assert len(v) == len(self.hypers), f"Trying to set {len(v)} values for {k} but there are {len(self.param_lists)} parameter groups."
         self._set_hyper(k, v)
 
+    def unfreeze(self): self.freeze_to(0)
```

Comment:
Missing docstring. Note `freeze` is also missing docstring.

Reply:
Should we add docstring if there is already `add_docs(_BaseOptimizer, ...)` for `freeze` and `unfreeze`?